sklearn/tests/test_impute.py

```diff
@@ -705,3 +705,23 @@ def test_chained_imputer_additive_matrix():
                              random_state=rng).fit(X_train)
     X_test_est = imputer.transform(X_test)
     assert_allclose(X_test_filled, X_test_est, atol=0.01)
+
+
+@pytest.mark.parametrize("imputer_constructor",
+                         [SimpleImputer, ChainedImputer])
+@pytest.mark.parametrize(
+    "missing_values, X_missing_value, err_msg",
+    [("NaN", np.nan, "contains"),
```

Comment:
Can you give more context for the contains error message? I don't know what the expected error is.

Reply:
It's the error raised by `check_array` when `X` contains `NaN` or `inf` and `force_all_finite=True:
```
Input contains NaN, infinity or a value too large for dtype('float64').
```

Reply:
right. would be easier to follow if it was "Input contains NaN" ...